bin/eslint.js

```diff
@@ -8,10 +8,11 @@ process.chdir(CODE_DIR);
 var stdout = console.log;
 console.log = console.error;
 
-var eslint = require('../lib/eslint-patch')(require('eslint'));
+
+var eslint = require('../lib/eslint-patch')();
 
 var CLIEngine = eslint.CLIEngine;
-var docs = eslint.docs;
```

Comment:
@GordonDiggs @maxjacobson I felt this was misplaced and was impacting the way we load/patch eslint, so I decoupled it from `eslint` instance. Haven't found any other use of `eslint.docs` so I think we are good, but wanted to hight light this.